npc/events/christmas_2006.txt

```diff
@@ -0,0 +1,4165 @@
+		next;
+		mes "[Mas]";
+		mes "Well then, I'll meet you 'there'.";
+		05christ_solo = 12;
+		close;
+	} else if (05christ_solo > 11) {
+		mes "[Mas]";
+		mes "^0000ffWhere the sun goes down^000000!";
+		next;
+		mes "[Mas]";
+		mes "I'm sorry, that's all I know.Go ask other members.";
+		next;
+		mes "[Mas]";
+		mes "Well then, I'll meet you 'there'.";
+		close;
+	} else {
+		mes "[Mas]";
+		mes "Herbs,potions.I can make it all by myself!! All I need is an honour.";
+		close;
+	}
+}
+
+prontera,35,209,5	script	Event#06xmas_01	4_M_SITDOWN,{
+	if (05christ_solo == 12) {
+		mes "[Event]";
+		mes "........................................I love you too, Miss Kafra!!!";
+		emotion ET_HUK, playerattached();
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "(Oh, shocks!)";
+		next;
+		mes "[Event]";
+		mes "You know what,Tinybee? Succubus hit on me the other.I hate being hansome!!";
+		next;
+		mes "-He starts talking to his right hand with a smile on his face.-";
+		emotion ET_KIK;
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "Well..Mr.Event...Is there anything you heard from Enjoy??...Hello!!Are you listening?!";
+		mes "(-Shuddering-)";
+		emotion ET_CRY, playerattached();
+		next;
+		mes "[Event]";
+		mes "Did you hear what she said Tinybee?She just said The Kafra services are always on your side.I know she just shy to say the truth. Why can't she just say stay Mr.Event~.";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "Excuse me... um...I believe Enjoy left a message for me...didn't he?...";
+		mes "(Ah~Such a weirdo!!!)";
+		next;
+		mes "[Event]";
+		mes "^0000ffAnother note^000000.";
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "What did you say??";
+		next;
+		mes "[Event]";
+		mes "^0000ffAnother note^000000.";
+		next;
+		mes "[Event]";
+		mes "Haha...I know what..You are trying to hear my voice again!Aren't you?...";
+		next;
+		mes "[Event]";
+		mes "Well then, I'll meet you 'there'.";
+		05christ_solo = 13;
+		close;
+	} else if (05christ_solo > 12) {
+		mes "[Event]";
+		mes "^0000ffAnother note^000000.";
+		next;
+		mes "[Event]";
+		mes "Haha...I know what..You are trying to hear my voice again!Aren't you?...";
+		next;
+		mes "[Event]";
+		mes "Well then, I'll meet you 'there'.";
+		close;
+	} else {
+		mes "[Event]";
+		mes "........................................I love you too, Miss Kafra!!!";
+		emotion ET_HUK, playerattached();
+		next;
+		mes "["+ strcharinfo(0) +"]";
+		mes "(Oh, shocks!)";
+		next;
+		mes "[Event]";
+		mes "You know what,Tinybee? Succubus hit on me the other.I hate being hansome!!";
+		next;
+		mes "-He starts talking to his right hand with a smile on his face.-";
+		emotion ET_KIK;
+		close;
+	}
+}
+
+//============================================================
+//= Event 5: Nightmare on Christmas Day
+//============================================================
+
+xmas,152,290,3	script	Poe#06xmas_poe01	4_M_KID2,{
+	if (06christ_santa < 1) {
```

Comment:
`.@name$ = strcharinfo(0);` to avoid the redundant command call

Reply:
Applied suggested fix.